src/domUpdates.js

```diff
@@ -36,8 +36,8 @@ const domUpdates = {
     })
     return `
     <div class="trip-card ${trip.status}">
-      <h5>Where are you headed?</h5>
-      <h6 class = "country-title"> ${destination.destination} </h6>
+      <h4>Where are you headed?</h4>
```

Comment:
nice change to bigger text

Reply:
Strangely enough this was just to make sure I had consecutive header elements, which I am still a little blurry on the accessibility benefits of.